less/index.less

```diff
@@ -61,9 +61,31 @@ html, body {
 
 // mixins
 
+.font(){
+	color:#333333;
+	font-family: 'Playfair Display', serif;
+
+}
+
+.header-font(){
+	color:#C2DFFF;
+	font-family: 'Playfair Display', serif;
+
+}
+
 // variables
 
+// container
 
+.container{
+	@media(max-width:500px){
```

Comment:
I think you forgot to add the viewport meta tag in your html file. It won't work with the chrome emulator when you try to test in on tablet/mobile view. 😄